Kickstarter-iOS/Features/ConfirmDetails/Controllers/ConfirmDetailsViewController.swift

```diff
@@ -0,0 +1,282 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public enum ConfirmDetailsLayout {
+  enum Margin {
+    static let topBottom: CGFloat = Styles.grid(3)
+    static let leftRight: CGFloat = CheckoutConstants.PledgeView.Inset.leftRight
+  }
+}
+
+protocol ConfirmDetailsViewControllerDelegate: AnyObject {}
+
+final class ConfirmDetailsViewController: UIViewController {
+  // MARK: - Properties
+
+  public weak var delegate: ConfirmDetailsViewControllerDelegate?
+
+  private lazy var titleLabel = UILabel(frame: .zero)
+
+  /// The pledge stepper used to change the pledge amount
+  private lazy var pledgeAmountViewController = {
+    PledgeAmountViewController.instantiate()
+      |> \.delegate .~ self
+  }()
+
+  /// The shipping location shown when changing shipping locations isn't an option
+  private lazy var shippingSummaryView: PledgeShippingSummaryView = {
+    PledgeShippingSummaryView(frame: .zero)
+  }()
+
+  /// The bottom-up modal for selecting a new shipping location
+  private lazy var shippingLocationViewController = {
+    PledgeShippingLocationViewController.instantiate()
+      |> \.delegate .~ self
+  }()
+
+  private lazy var localPickupLocationView = {
+    PledgeLocalPickupView(frame: .zero)
+  }()
+
+  private lazy var inputsSectionViews = {
+    [
+      self.shippingLocationViewController.view,
+      self.shippingSummaryView,
+      self.localPickupLocationView,
+      self.pledgeAmountViewController.view
+    ]
+  }()
+
+  private lazy var keyboardDimissingTapGestureRecognizer: UITapGestureRecognizer = {
+    UITapGestureRecognizer(
+      target: self,
+      action: #selector(ConfirmDetailsViewController.dismissKeyboard)
+    )
+      |> \.cancelsTouchesInView .~ false
```

Comment:
Nit: I would prefer to see these done without the use of Prelude/lenses, just because I have a sneaking suspicion (based on some suspicious timing output in Xcode) that the the use of many Prelude operators leads to slow compile times. Using lenses for mutable properties like `cancelsTouchesInView` is unecessary. 

However, I agree that being expedient is the most important thing here, so if this is the fastest way for you to do this - I assume you're working off an example elsewhere in the app - ignore me.